.github/autobuild/mac.sh

```diff
@@ -30,20 +30,35 @@ prepare_signing() {
     [[ -n "${MACOS_CERTIFICATE:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_ID:-}" ]] || return 1
     [[ -n "${MACOS_CERTIFICATE_PWD:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE_ID:-}" ]] || return 1
+    [[ -n "${MACAPP_CERTIFICATE_PWD:-}" ]] || return 1
     [[ -n "${NOTARIZATION_PASSWORD:-}" ]] || return 1
     [[ -n "${KEYCHAIN_PASSWORD:-}" ]] || return 1
 
     echo "Signing was requested and all dependencies are satisfied"
 
-    # Put the cert to a file
-    echo "${MACOS_CERTIFICATE}" | base64 --decode > certificate.p12
+    ## Put the certs to a file
+    # MACOS_CERTIFICATE - Mac Installer Distribution
+    # MACAPP_CERTIFICATE - Mac App Distribution
```

Comment:
I think you should put this comment a few lines above to show the difference between MACOS_CERTIFICATE and MACAPP_CERTIFICATE. 
Honestly, I'd prefer something clearer (something with deployment or distribution)

Reply:
Thanks for picking up on this :)
I didn't want to break the existing MACOS_CERTIFICATE but ideally I agree, it should be something like:
MAC_INST_DIST_CERT - for Mac Installer Distribution certificate
MAC_APP_DIST_CERT - for Mac App Distribution certificate

and make the Github Secrets names correspond appropriately.
You also need to store a password and the ID for each certificate (though you can re-use the same password for both certs)

How it could possibly look in `autobuild.yml`:

```
env:
          JAMULUS_BUILD_VERSION:       ${{ needs.create_release.outputs.build_version }}
          MAC_INST_DIST_CERT:               ${{ secrets.MAC_INST_DIST_CERT}}
          MAC_INST_DIST_CERT_PWD:    ${{ secrets.MAC_INST_DIST_CERT_PWD }}
          MAC_INST_DIST_CERT_ID:         ${{ secrets.MAC_INST_DIST_CERT_ID }}
          MAC_APP_DIST_CERT:                ${{ secrets.MAC_APP_DIST_CERT}}
          MAC_APP_DIST_CERT_PWD:     ${{ secrets.MAC_APP_DIST_CERT_PWD }}
          MAC_APP_DIST_CERT_ID:          ${{ secrets.MAC_APP_DIST_CERT_ID }}
```